services/catalog/src/simcore_service_catalog/api/routes/services.py

```diff
@@ -188,11 +194,12 @@ async def cached_registry_services() -> dict[str, Any]:
 
 
 SIMCORE_SERVICE_SETTINGS_LABELS: Final[str] = "simcore.service.settings"
+SIMCORE_SERVICE_COMPOSE_SPEC_LABEL: Final[str] = "simcore.service.compose-spec"
 
 
 @router.get(
     "/{service_key:path}/{service_version}/resources",
-    response_model=Dict[ResourceName, ResourceValue],
+    response_model=ServiceResources,
```

Comment:
Again, if you write here ``dict[ResrouceName, ResourveValue`` (as was before) , all your problems with fastapi are gone. :-)